packages/plasma-new-hope/src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -18,7 +18,9 @@ export const breadcrumbsRoot = (Root: RootProps<HTMLDivElement, BreadcrumbsProps
 
             return (
                 <Root ref={outerRootRef} size={size} view={view} className={cx(className)} items={items}>
-                    {itemsWithSeparator.map((item) => item)}
+                    {itemsWithSeparator.map((item, index) => (
+                        <React.Fragment key={index}>{item}</React.Fragment>
+                    ))}
```

Comment:
Зачем тут вообще запариваться с кеями?
```suggestion
                    {itemsWithSeparator}
```

Reply:
пушто реакт в консоль красными ворнингами иначе срёт:

<img width="1096" alt="image" src="https://github.com/user-attachments/assets/6731a041-1408-4310-9adb-ac81ef03f74d">